src/memory/paddr.c

```diff
@@ -19,6 +19,7 @@
 #include <memory/paddr.h>
 #include <memory/store_queue_wrapper.h>
 #include <memory/sparseram.h>
+#include <device/map.h>
```

Comment:
Is this header include required? It seems other lines are unchanged.

Reply:
i'll delete it